docs/blog/2017-07-19-creating-a-blog-with-gatsby/index.md

```diff
@@ -427,11 +427,12 @@ exports.createPages = ({ actions, graphql }) => {
         }
       }
     }
-  `).then(result => {
-    if (result.errors) {
-      return Promise.reject(result.errors)
-    }
-  })
+  `)
+
+  if (result.errors) {
+    reporter.panic(`Error while running GraphQL query.`)
```

Comment:
Using panic on build is properly better so users can use graphiql to explore their mistakes as @LekoArts suggested.
```suggestion
    reporter.panicOnBuild(`Error while running GraphQL query.`)
```